BookStore.h

```diff
@@ -0,0 +1,135 @@
+#ifndef UNTITLED_BOOKSTORE_H
+#define UNTITLED_BOOKSTORE_H
+#ifndef UNTITLED_H
+#define UNTITLED_H
+
+#include "string"
+#include "vector"
+#include "algorithm"
+#include "iostream"
+
+class Book {
+public:
+    Book(std::string name,
+         int price_,
+         int numberOfPages_,
+         std::string author_,
+         int quantity_,
+         int numberOfSales_);
```

Comment:
Weird project structure. Although this is supposed to be a header file for the class BookStore, it includes the code that must be separated into 4 different files(Bookstore .cpp/.h, Book.cpp/.h). The project already has BookStore.cpp file but it  is not used for its purpose, currently is used as a main.cpp that tests both classes.

Reply:
divided into Bookstore.h, Book.h and main.cpp